### PR TITLE
Build scripts for better-ajv-errors

### PR DESCRIPTION
**Pull request by Mahima-Gaikwad**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: b/better-ajv-errors/LICENSE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright 2018 Atlassian Pty Ltd
```
Comment (seth-priya) [inline b/better-ajv-errors/LICENSE:1]: this is the script license and not the package license, can just be copied over from the base folder of this repo

File: b/better-ajv-errors/better-ajv-errors_ubi8.4.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : better-ajv-errors
+# Version       : v0.5.7
+# Source repo   : https://github.com/atlassian/better-ajv-errors.git
+# Tested on     : UBI 8.4
+# Script License: Apache 2.0 License
+# Maintainer    : Mahima Gaikwad <mahima.gaikwad@ibm.com>
+#
```
Comment (seth-priya) [inline b/better-ajv-errors/better-ajv-errors_ubi8.4.sh:9]: Please add the additional fields that have been introduced recently - Language, Travis-Check etc

File: b/better-ajv-errors/better-ajv-errors_ubi8.4.sh [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+set -e
+
+#Variables
+PACKAGE_URL=https://github.com/atlassian/better-ajv-errors.git
+PACKAGE_VERSION=v0.5.7
+PACKAGE=better-ajv-errors
```
Comment (seth-priya) [inline b/better-ajv-errors/better-ajv-errors_ubi8.4.sh:25]: use PACKAGE_NAME for better clarity

File: b/better-ajv-errors/better-ajv-errors_ubi8.4.sh [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+NODE_VERSION=v12.22.8
+DISTRO=linux-ppc64le
+
+yum update -y 
+
+#Install dev dependencies 
+yum install -y git wget gcc gcc-c++ python2 make
```
Comment (seth-priya) [inline b/better-ajv-errors/better-ajv-errors_ubi8.4.sh:32]: are all of these dependencies needed?

File: b/better-ajv-errors/better-ajv-errors_ubi8.4.sh [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+set -e
+
+#Variables
+PACKAGE_URL=https://github.com/atlassian/better-ajv-errors.git
+PACKAGE_VERSION=v0.5.7
+PACKAGE=better-ajv-errors
+NODE_VERSION=v12.22.8
+DISTRO=linux-ppc64le
+
+yum update -y
```
Comment (seth-priya) [inline b/better-ajv-errors/better-ajv-errors_ubi8.4.sh:29]: do we need a yum update?

File: c/call-bind/LICENSE [hunk 6]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (seth-priya) [inline c/call-bind/LICENSE:1]: same comment as above - the LICENSE can just be copied over from the base folder

File: c/call-bind/call-bind_ubi8.4.sh [hunk 7]
```diff
@@ -0,0 +1,51 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : call-bind
+# Version       : master(a994df69f401f4bf735a4ccd77029b85d1549453)
```
Comment (seth-priya) [inline c/call-bind/call-bind_ubi8.4.sh:4]: let's say Commit # : <commit id>

File: c/call-bind/call-bind_ubi8.4.sh [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+# Source repo   : https://github.com/ljharb/call-bind.git
+# Tested on     : UBI 8.4
+# Script License: MIT License
```
Comment (seth-priya) [inline c/call-bind/call-bind_ubi8.4.sh:7]: this is script license, not the package license, should be Apache 2.0

File: c/call-bind/call-bind_ubi8.4.sh [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+set -e
+
+#Variables
+PACKAGE_URL=https://github.com/ljharb/call-bind.git
+PACKAGE=call-bind
```
Comment (seth-priya) [inline c/call-bind/call-bind_ubi8.4.sh:24]: use "PACKAGE_NAME" for clarity

File: c/call-bind/call-bind_ubi8.4.sh [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+#Test fails for required version and latest stable version
+PACKAGE_VERSION=a994df69f401f4bf735a4ccd77029b85d1549453
+NODE_VERSION=v12.22.8
+DISTRO=linux-ppc64le
+
+yum update -y 
+
+#Install dev dependencies 
+yum install -y git wget gcc gcc-c++ python2 make
```
Comment (seth-priya) [inline c/call-bind/call-bind_ubi8.4.sh:33]: are all of these needed?

File: c/call-bind/call-bind_ubi8.4.sh [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+#Variables
+PACKAGE_URL=https://github.com/ljharb/call-bind.git
+PACKAGE=call-bind
+#Test fails for required version and latest stable version
+PACKAGE_VERSION=a994df69f401f4bf735a4ccd77029b85d1549453
+NODE_VERSION=v12.22.8
+DISTRO=linux-ppc64le
+
+yum update -y
```
Comment (seth-priya) [inline c/call-bind/call-bind_ubi8.4.sh:30]: is this needed?

**Review by seth-priya — CHANGES_REQUESTED** [review]

**Comment by seth-priya** [comment]
Generic question - any specific reason for not checking " Have you checked and followed all the points mention in the CONTRIBUTING.MD" above?

File: b/better-ajv-errors/better-ajv-errors_ubi8.4.sh (hunk 2 above)
Comment (Mahima-Gaikwad) [reply b/better-ajv-errors/better-ajv-errors_ubi8.4.sh:11]: Added Language, Travis-Check fields

**Comment by Mahima-Gaikwad** [comment]
Updated builds scripts as per above comments.
Updated License file, replaced package License with script License.
Checked "Have you checked and followed all the points mention in the CONTRIBUTING.MD"

File: b/better-ajv-errors/LICENSE (hunk 1 above)
Comment (Mahima-Gaikwad) [reply b/better-ajv-errors/LICENSE]: Updated license file

File: b/better-ajv-errors/better-ajv-errors_ubi8.4.sh (hunk 3 above)
Comment (Mahima-Gaikwad) [reply b/better-ajv-errors/better-ajv-errors_ubi8.4.sh]: used PACKAGE_NAME

File: b/better-ajv-errors/better-ajv-errors_ubi8.4.sh (hunk 5 above)
Comment (Mahima-Gaikwad) [reply b/better-ajv-errors/better-ajv-errors_ubi8.4.sh]: Removed yum update

File: b/better-ajv-errors/better-ajv-errors_ubi8.4.sh (hunk 4 above)
Comment (Mahima-Gaikwad) [reply b/better-ajv-errors/better-ajv-errors_ubi8.4.sh]: Dependencies not needed, removed

File: c/call-bind/call-bind_ubi8.4.sh (hunk 8 above)
Comment (Mahima-Gaikwad) [reply c/call-bind/call-bind_ubi8.4.sh]: Updated License

File: c/call-bind/call-bind_ubi8.4.sh (hunk 7 above)
Comment (Mahima-Gaikwad) [reply c/call-bind/call-bind_ubi8.4.sh]: Updated commit id

File: b/better-ajv-errors/better-ajv-errors_ubi8.4.sh (hunk 3 above)
Comment (Mahima-Gaikwad) [reply b/better-ajv-errors/better-ajv-errors_ubi8.4.sh]: used PACKAGE_NAME

**Review by seth-priya — APPROVED** [review]

**Pull request merged**